FiveCalls/FiveCalls/Contact.swift

```diff
@@ -32,6 +32,17 @@ struct Contact {
     let photoURL: URL?
     let reason: String
     let state: String
+    var hasContacted: Bool
+}
+
+extension Contact: Hashable {
```

Comment:
Yeah I forgot to pull these, I made it Hashable to put it in a Set but removed that code, but yes would make it Equatable as well.